src/graphql/schema.js

```diff
@@ -418,19 +418,19 @@ type Query {
     skip: Int
   ): PaginatedBiggestWinner!
 
-  eventLeaderboard(
-    filter: LeaderboardFilter
+  eventLeaderboardEntries(
```

Comment:
This should still be `eventLeaderboard`.  Sorry wasn't clear. The `eventLeaderboard` query returns a list of `LeaderboardEntry`s. Like a `users` query will return a list of `User`s. 

The relation is one-to-many. One `eventLeaderboard` query will return many `LeaderboardEntry`s.

Reply:
Think `eventLeaderboardEntries` works better since we are querying for a set of paginated entries not the whole leaderboard.

In Event store, we only query for one specific entry just to get the winnings.

Reply:
Fair enough. Actually it does conform to the other query names.